docs/hub/asteroid.md

```diff
@@ -7,8 +7,8 @@
 You can find `asteroid` models by filtering at the left of the [models page](https://huggingface.co/models?filter=asteroid). 
 
 All models on the Hub come up with the following features:
-1. An automatically generated model card with a description, a training configuration, metrics and more.
-2. Metadata tags that help for discoverability and contain information such as license and datasets.
+1. An automatically generated model card with a description, training configuration, metrics, and more.
+2. Metadata tags that help for discoverability and contain information such as licenses and datasets.
```

Comment:
```suggestion
2. Metadata tags that help for discoverability and contain information such as the license and datasets.
```

Reply:
In general, a single license is preferred so probably makes sense to keep this singular :)